src/react/src/components/Dashboard/DashboardModerationQueue.jsx

```diff
@@ -65,6 +75,8 @@ const DashboardModerationQueue = ({
     const wasNotEmptyAndNowEmpty = (prev, current) =>
         prev?.current?.length > 0 && current?.length === 0;
 
+    const isValidDate = date => !date || dateRegexFormat.test(date);
```

Comment:
I suggest to use this validation:
`const isValidDate = date => typeof date === 'string' && dateRegexFormat.test(date);`
It ensures that input is string and rejects `null`, `undefined`, `numbers`, etc.

Reply:
Done.

Reply:
Also a litle bit reworked due to requirements